src/electionguard/guardian.py

```diff
@@ -154,7 +154,13 @@ def __init__(
         sequence_order: int,
         number_of_guardians: int,
         quorum: int,
-        nonce_seed: Optional[ElementModQ] = None,
+        election_keys: ElectionKeyPair = None,
```

Comment:
Refactor to `key_pair`. This field should be required.

Reply:
Will refactor rename, but it cannot be made required without significant modifications because the vast majority of instantiations of Guardian do not provide a key pair.